sql: TestShowQueries failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/d957838032a2dc162794efd23432f2672322f231

Parameters:
```
TAGS=
GOFLAGS=
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=837282&tab=buildLog

```
=== RUN   TestShowQueries
W180815 05:15:50.916357 154095 server/status/runtime.go:287  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180815 05:15:50.921429 154095 server/server.go:827  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180815 05:15:50.921546 154095 base/addr_validation.go:260  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 05:15:50.921566 154095 base/addr_validation.go:300  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 05:15:50.923093 154095 server/config.go:496  [n?] 1 storage engine initialized
I180815 05:15:50.923118 154095 server/config.go:499  [n?] RocksDB cache size: 128 MiB
I180815 05:15:50.923127 154095 server/config.go:499  [n?] store 0: in-memory, size 0 B
I180815 05:15:50.929001 154095 server/node.go:373  [n?] **** cluster 7b9940b5-f59e-4752-b6e1-8af3934c4ee0 has been created
I180815 05:15:50.929022 154095 server/server.go:1398  [n?] **** add additional nodes by specifying --join=127.0.0.1:33747
I180815 05:15:50.929154 154095 gossip/gossip.go:383  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:33747" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:11 > build_tag:"v2.1.0-alpha.20180702-1564-gd957838" started_at:1534310150929092703 
I180815 05:15:50.931412 154095 storage/store.go:1506  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180815 05:15:50.931498 154095 server/node.go:476  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=6.9 KiB), ranges=1, leases=0, queries=0.00, writes=0.00, bytesPerReplica={p10=7103.00 p25=7103.00 p50=7103.00 p75=7103.00 p90=7103.00 pMax=7103.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180815 05:15:50.931544 154095 storage/stores.go:222  [n1] read 0 node addresses from persistent storage
I180815 05:15:50.931612 154095 server/node.go:697  [n1] connecting to gossip network to verify cluster ID...
I180815 05:15:50.932959 154095 server/node.go:722  [n1] node connected via gossip and verified as part of cluster "7b9940b5-f59e-4752-b6e1-8af3934c4ee0"
I180815 05:15:50.932990 154095 server/node.go:546  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180815 05:15:50.933203 154095 server/status/recorder.go:652  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 05:15:50.933222 154095 server/server.go:1831  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180815 05:15:50.933276 154095 server/server.go:1535  [n1] starting https server at 127.0.0.1:34333 (use: 127.0.0.1:34333)
I180815 05:15:50.933290 154095 server/server.go:1537  [n1] starting grpc/postgres server at 127.0.0.1:33747
I180815 05:15:50.933302 154095 server/server.go:1538  [n1] advertising CockroachDB node at 127.0.0.1:33747
I180815 05:15:50.934730 154367 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 05:15:50.949972 154418 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180815 05:15:50.958994 154434 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:root}
I180815 05:15:50.964695 154440 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
I180815 05:15:50.987556 154404 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:gen_random_uuid()::STRING User:root}
I180815 05:15:50.993898 154410 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180815 05:15:50.997843 154414 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180815 05:15:51.006960 154095 server/server.go:1620  [n1] done ensuring all necessary migrations have run
I180815 05:15:51.006991 154095 server/server.go:1623  [n1] serving sql connections
I180815 05:15:51.009193 154468 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180815 05:15:51.011292 154470 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:33747} Attrs: Locality: ServerVersion:2.0-11 BuildTag:v2.1.0-alpha.20180702-1564-gd957838 StartedAt:1534310150929092703 LocalityAddress:[]} ClusterID:7b9940b5-f59e-4752-b6e1-8af3934c4ee0 StartedAt:1534310150929092703 LastUp:1534310150929092703}
W180815 05:15:51.015217 154095 server/status/runtime.go:287  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180815 05:15:51.018977 154095 server/server.go:827  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180815 05:15:51.019090 154095 base/addr_validation.go:260  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 05:15:51.019110 154095 base/addr_validation.go:300  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 05:15:51.021263 154095 server/config.go:496  [n?] 1 storage engine initialized
I180815 05:15:51.021279 154095 server/config.go:499  [n?] RocksDB cache size: 128 MiB
I180815 05:15:51.021287 154095 server/config.go:499  [n?] store 0: in-memory, size 0 B
W180815 05:15:51.021334 154095 gossip/gossip.go:1351  [n?] no incoming or outgoing connections
I180815 05:15:51.021364 154095 server/server.go:1400  [n?] no stores bootstrapped and --join flag specified, awaiting init command.
I180815 05:15:51.033332 154513 gossip/client.go:129  [n?] started gossip client to 127.0.0.1:33747
I180815 05:15:51.040361 154529 gossip/server.go:219  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:46359}
I180815 05:15:51.041094 154095 server/node.go:697  [n?] connecting to gossip network to verify cluster ID...
I180815 05:15:51.041138 154095 server/node.go:722  [n?] node connected via gossip and verified as part of cluster "7b9940b5-f59e-4752-b6e1-8af3934c4ee0"
I180815 05:15:51.041491 154552 kv/dist_sender.go:345  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180815 05:15:51.051960 154551 kv/dist_sender.go:345  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180815 05:15:51.052607 154095 kv/dist_sender.go:345  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180815 05:15:51.053310 154095 server/node.go:428  [n?] new node allocated ID 2
I180815 05:15:51.053370 154095 gossip/gossip.go:383  [n2] NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:46359" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:11 > build_tag:"v2.1.0-alpha.20180702-1564-gd957838" started_at:1534310151053316888 
I180815 05:15:51.053430 154095 storage/stores.go:222  [n2] read 0 node addresses from persistent storage
I180815 05:15:51.053457 154095 storage/stores.go:241  [n2] wrote 1 node addresses to persistent storage
I180815 05:15:51.054287 154460 storage/stores.go:241  [n1] wrote 1 node addresses to persistent storage
I180815 05:15:51.055269 154095 server/node.go:672  [n2] bootstrapped store [n2,s2]
I180815 05:15:51.055370 154095 server/node.go:546  [n2] node=2: started with [] engine(s) and attributes []
I180815 05:15:51.055580 154095 server/status/recorder.go:652  [n2] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 05:15:51.055603 154095 server/server.go:1831  [n2] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180815 05:15:51.055677 154095 server/server.go:1535  [n2] starting https server at 127.0.0.1:37861 (use: 127.0.0.1:37861)
I180815 05:15:51.055691 154095 server/server.go:1537  [n2] starting grpc/postgres server at 127.0.0.1:46359
I180815 05:15:51.055703 154095 server/server.go:1538  [n2] advertising CockroachDB node at 127.0.0.1:46359
I180815 05:15:51.056605 154095 server/server.go:1620  [n2] done ensuring all necessary migrations have run
I180815 05:15:51.056633 154095 server/server.go:1623  [n2] serving sql connections
I180815 05:15:51.057245 154650 server/status/recorder.go:652  [n2,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 05:15:51.067677 154557 sql/event_log.go:126  [n2] Event: "node_join", target: 2, info: {Descriptor:{NodeID:2 Address:{NetworkField:tcp AddressField:127.0.0.1:46359} Attrs: Locality: ServerVersion:2.0-11 BuildTag:v2.1.0-alpha.20180702-1564-gd957838 StartedAt:1534310151053316888 LocalityAddress:[]} ClusterID:7b9940b5-f59e-4752-b6e1-8af3934c4ee0 StartedAt:1534310151053316888 LastUp:1534310151053316888}
I180815 05:15:51.076232 154000 sql/event_log.go:126  [n1,client=127.0.0.1:37224,user=root] Event: "create_database", target: 52, info: {DatabaseName:test Statement:CREATE DATABASE IF NOT EXISTS test User:root}
I180815 05:15:51.079326 154000 sql/event_log.go:126  [n1,client=127.0.0.1:37224,user=root] Event: "create_table", target: 53, info: {TableName:test.public.aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa💩aaaaaaaaaaaaaaaaa Statement:CREATE TABLE test.public.aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa💩aaaaaaaaaaaaaaaaa (num INT) User:root}
I180815 05:15:51.099101 154488 rpc/nodedialer/nodedialer.go:92  [intExec=get-table-statistics] connection to n1 established
I180815 05:15:51.118277 154763 rpc/nodedialer/nodedialer.go:92  [n1] connection to n2 established
I180815 05:15:51.129631 154555 server/server_update.go:67  [n2] no need to upgrade, cluster already at the newest version
--- FAIL: TestShowQueries (0.24s)
	show_test.go:569: unexpected query in SHOW QUERIES: "SHOW CLUSTER SETTING version", expected: "SELECT * FROM aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\u2026"

```

--- a/pkg/sql/show_test.go
+++ b/pkg/sql/show_test.go
@@ -507,6 +507,7 @@ func TestShowQueries(t *testing.T) {
 		if stmt == selectStmt {
 			found = true
 			const showQuery = "SELECT node_id, (now() - start)::FLOAT, query FROM [SHOW CLUSTER QUERIES]"
+			const showVersionQuery = "SHOW CLUSTER SETTING VERSION"
 
 			rows, err := conn1.Query(showQuery)
 			if err != nil {
@@ -526,7 +527,7 @@ func TestShowQueries(t *testing.T) {
 					return
 				}
 				switch sql {
-				case showQuery, expectedSelectStmt:
+				case showQuery, showVersionQuery, expectedSelectStmt:
 				default:
 					failure = fmt.Errorf(
 						"unexpected query in SHOW QUERIES: %+q, expected: %+q",

